deploy/olm-catalog/eclipse-che-preview-kubernetes/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -215,15 +215,15 @@ spec:
                       - name: RELATED_IMAGE_che_tls_secrets_creation_job
                         value: quay.io/eclipse/che-tls-secret-creator:alpine-d1ed4ad
                       - name: RELATED_IMAGE_pvc_jobs
-                        value: registry.access.redhat.com/ubi8-minimal:8.2-345
+                        value: registry.access.redhat.com/ubi8-minimal:8.2-349
                       - name: RELATED_IMAGE_postgres
                         value: centos/postgresql-96-centos7:9.6
                       - name: RELATED_IMAGE_keycloak
                         value: quay.io/eclipse/che-keycloak:nightly
                       - name: RELATED_IMAGE_che_workspace_plugin_broker_metadata
-                        value: quay.io/eclipse/che-plugin-metadata-broker:v3.4.0
+                        value: quay.io/eclipse/che-plugin-metadata-broker:v3.3.0
                       - name: RELATED_IMAGE_che_workspace_plugin_broker_artifacts
-                        value: quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0
+                        value: quay.io/eclipse/che-plugin-artifacts-broker:v3.3.0
```

Comment:
Current broker version is `3.4.0`. This change will make https://github.com/eclipse/che/pull/17785 unmergeable.

Reply:
@amisevsk 
I see. There is misunderstood of how images are updated in the operator.yaml.
We synchronize images with che.properties on releasing major version of the operator.

So, the correct workflow should be:
1. Update che.properties
2. Update operator.yaml or wait until releasing major version of the che-operator

In our case the solution might be:
1. https://github.com/eclipse/che/pull/17785 is merged before 7.19.0 then I update my PR correspondingly 
2. https://github.com/eclipse/che/pull/17785 isn't merged before 7.19.0 then I have to merge my PR to synchronize images with che.properties.

Reply:
Ah I understand now, thanks for explaining. The reason I updated here is that this flow makes it hard to actually merge the entire set of changes:
- The Che server changes require updating the broker; workspaces fail to start with Che server changes but broker `v3.3.0`
- The Operator overrides the Che value in happy path tests, so until the operator is using broker `v3.4.0`, the Che server changes are unmergeable

In effect, the operator prevents merging changes on the Che side, and the Che side prevents merging changes on the operator side. I suppose the way to get around this is to have a separate PR that updates just the property, followed by a PR that adds the other changes, but this could be a problem in the future.

Reply:
@tolusha I've created https://github.com/eclipse/che/pull/17870, if we merge this very soon, will that resolve the issue?

Reply:
yes, the issue will be resolved.  I will update my PR correspondingly. 
thank you!